torch/distributed/checkpoint/format_utils.py

```diff
@@ -81,3 +76,21 @@ def dcp_to_torch_save(
         no_dist=True,
     )
     torch.save(sd, torch_save_fn)
+
+
+def torch_save_to_dcp(
+    torch_save_fn: Union[str, os.PathLike], dcp_checkpoint_dir: Union[str, os.PathLike]
```

Comment:
Same as the previous PR, we can avoid using `fn` to represent function name. We can use `path`.